example.env

```diff
@@ -173,6 +173,9 @@ GOTRUE_SMS_TEXTLOCAL_SENDER=""
 GOTRUE_SMS_VONAGE_API_KEY=""
 GOTRUE_SMS_VONAGE_API_SECRET=""
 GOTRUE_SMS_VONAGE_FROM=""
+GOTRUE_SMS_CUSTOM_URL=""
+GOTRUE_SMS_CUSTOM_SENDER=""
+GOTRUE_SMS_CUSTOM_BEARER_TOKEN=""
```

Comment:
`SENDER` is not necessary IMO, while the `BEARER_TOKEN` should be replaced with a JSON-like map that sets various headers on the request (not just Authorization).

Reply:
I know `SENDER` is not necessary but supplying it aligns better with other provider configurations.